### ISSUE TITLE
Moderators Need Option to Refuse Moderation

### ISSUE THREAD

**Issue by SamPatt** [OpenBazaar/OpenBazaar-Client#1002]
If a moderator is selected for a transaction that is illegal or they don't want to moderate for other reasons, a moderator needs the ability to formally reject the transaction.

This likely means that vendors and buyers also need the ability to switch moderators; I'll open another issue for that.

**Comment by morebrownies** [comment]
At a higher level, a moderator should have the ability to accept/decline new vendor(s) before they're included on transactions.

**Comment by drwasho** [comment]
I don't think this can't be done on a per transaction basis, as that would involve 'asking permission' to moderator every transaction before a purchase can be made (which isn't ideal)... only asking when there's a dispute is a bit crappy from a UX perspective.

Perhaps we can make it so that Vendors have to Follow a Moderator?

**Comment by jjeffryes** [comment]
It might be good to give the moderator an option when they see a dispute in their cases view to decline to moderate, which would undo the dispute status of that transaction. 

They should also have an option in the same place to remove themselves as a moderator for that store, which should prevent the store from adding them again as a moderator.

And then we'd need a way for the buyer and seller to pick a new moderator.

This opens multiple cans of worms.

**Comment by SimonBelmond** [comment]
Moderation should for sure not "auto" start on a sale when the server of the mod is down, which is what happens now. A policy to set

[] Auto accept trades for moderation when online;
[] manually accept each moderation

would be good.
